### PR TITLE
fix(fonts): Show the all the A's in our workspace correctly, not funky

### PR DESCRIPTION
**Pull request by prosdev0107**
### SUMMARY
All the A's in our workspace buttons look funky

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
BEFORE:
image

AFTER:
image

### TESTING INSTRUCTIONS
1. Go to the Welcome page
2. You can show that the A's looks funky there.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "@emotion/cache": "^11.4.0",
     "@emotion/react": "^11.4.1",
     "@emotion/styled": "^11.3.0",
-    "@fontsource/inter": "^4.5.7",
+    "@fontsource/inter": "^4.0.0",
```
Comment (diegomedina248) [inline superset-frontend/package.json:83]: what caused the issue with the previous version of the lib?
Did you try the latest version too?

Comment (prosdev0107) [reply superset-frontend/package.json:83]: yes sure, it also gives issues with latest version and so I downgrade it into 4.0.0 version from [this PR](https://github.com/apache/superset/pull/19236)

**Review by diegomedina248 — APPROVED** [review]

**Review by kgabryje — APPROVED** [review]
Nice! Does the bug also occur with monospace font?

**Review by stephenLYZ — APPROVED** [review]

**Comment by rusackas** [comment]
> Nice! Does the bug also occur with monospace font?

Not that we've seen. I think it's specifically an Inter issue.

**Comment by prosdev0107** [comment]
> 

I believe that it doesn't make the issue with monospace font.

**Pull request merged**